py/desispec/desi_proc_dashboard.py

```diff
@@ -85,7 +85,7 @@ def parse(options):
     # File I/O
     parser.add_argument('--redux-dir', type=str, help="Product directory, point to $DESI_SPECTRO_REDUX by default ")
     parser.add_argument('--output-dir', type=str, help="output portal directory for the html pages, which defaults to your home directory ")
-    parser.add_argument('--output-name', type=str, default='dashboard.html', help="name of the html page (to be placed in --output-dir).")
+    parser.add_argument('--output-name', type=str, default='dashboard', help="name of the html page (to be placed in --output-dir).")
```

Comment:
Anthony identified the more important issue (crashing), but another minor point: let's keep this as the original method of specifying the output-name including the extension so that we don't arbitrarily break existing cronjobs, notes, etc. that expect to specify `--output-name dashboard.html`